### DISCUSSION TITLE
make a variable that is only visible to sprites in a folder

### DISCUSSION THREAD

**Discussion by theTWIXhunter** [ScratchAddons/ScratchAddons#6983] [Ideas]
I would like to have a variable that is only visible to sprites in a certain folder (see the sprite folder addon)

**Comment by CubesterYT** [comment]
This isn't possible, as we can't modify the VM to make this work, even if we could, it can't happen as folders are a visual gimmick, relying on renaming assets to include the folder name to group them, when in reality, they are left the same unless a user uses Scratch Addons along with the folder addon.
